test/examples/transactions.js

```diff
@@ -55,7 +56,8 @@ describe('examples(transactions):', function() {
       async function updateEmployeeInfo(client, session) {
         session.startTransaction({
           readConcern: { level: 'snapshot' },
-          writeConcern: { w: 'majority' }
+          writeConcern: { w: 'majority' },
+          readPreference: new ReadPreference('primary')
```

Comment:
```
readPreference: 'primary'
```
works just as well, and is imho more convenient for users than having to import a named type. I think it's our responsibility to make sure that if someone uses a string its properly wrapped in the type internally, but I'm not sure I see a compelling reason to require them to wrap it here

Reply:
does it actually work here? I can't actually remember where we resolve read preference, but since this is in core it bypasses a lot of the topology stuff.

Reply:
Yeah this ultimately plumbs down into [this code](https://github.com/mongodb-js/mongodb-core/blob/master/lib/wireprotocol/command.js#L23) which does the correct translation as needed.